packages/components/inputs/rich-text-utils/src/rich-text-body/dropdown.tsx

```diff
@@ -10,26 +14,69 @@ import {
   DropdownItem as StyledDropdownItem,
 } from './dropdown.styles';
 
-const getIsSelected = (props, item) =>
+type Props = {
```

Comment:
I guess it would be better to name this following the name of the component: `TDropdownProps`.